modules/viewing-stats-collected-kubernetes-nmtate-op.adoc

```diff
@@ -0,0 +1,90 @@
+// This is included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-stats-collected-kubernetes-nmtate-op_{context}"]
+= Viewing metrics collected by the Kubernetes NMState Operator
+
+The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and them as ready-to-use metrics.
+
+[NOTE]
+====
+The metrics `kubernetes_nmstate_features_applied` are not an API and might change between {product-title} versions.
+====
+
+In the *Developer* perspective, the Metrics UI includes some predefined CPU, memory, bandwidth, and network packet queries for the selected project. You can run custom Prometheus Query Language (PromQL) queries for CPU, memory, bandwidth, network packet and application metrics for the project.
+
+You can also obtain the HTP service endpoint by specifying the namespace for the `kubernetes_nmstate_features_applied` service. For example:
```

Comment:
I feel like this sentence does not make much sense here. What we are "exampling" is not obtaining a service endpoint but querying for a specific field